.github/workflows/ci.yml

```diff
@@ -132,6 +134,17 @@ jobs:
       project-name: OpenTelemetry.Exporter.Geneva
       code-cov-name: Exporter.Geneva
 
+  build-test-host:
+    needs: detect-changes
+    if: |
+      contains(needs.detect-changes.outputs.changes, 'host')
```

Comment:
"host" is too generic? Maybe something more narrow like ResourceDetector.host, if feasible.

Reply:
It is in sync with current pattern. Will be handled in scope of #1513 for all packages (for process we have already names conflict).